backend/src/openarchiefbeheer/destruction/tests/endpoints/test_co_reviews.py

```diff
@@ -0,0 +1,147 @@
+from django.urls import reverse
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from openarchiefbeheer.accounts.tests.factories import UserFactory
+from openarchiefbeheer.destruction.constants import ListRole, ListStatus
+from openarchiefbeheer.destruction.tests.factories import (
+    DestructionListAssigneeFactory,
+    DestructionListCoReviewFactory,
+    DestructionListFactory,
+)
+
+
+class CoReviewsViewSetTest(APITestCase):
+    def test_list_not_logged_in(self):
+        url = reverse("api:destruction-list-co-reviews-list")
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_list(self):
+        user = UserFactory()
+
+        self.client.force_login(user)
+        url = reverse("api:destruction-list-co-reviews-list")
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_list_filter_destruction_list__uuid_not_logged_in(self):
```

Comment:
I think that this test is not needed, it tests the same as `test_list_not_logged_in`?